app/src/main/java/org/mozilla/focus/home/pocket/PocketVideoFragment.kt

```diff
@@ -53,7 +54,7 @@ class PocketVideoFragment : Fragment() {
         // being reconstructed, the user probably cares more about an up-to-date feed rather than
         // consistency with the home screen they saw a while ago.
         if (!this::deferredVideos.isInitialized) {
-            deferredVideos = Pocket.getRecommendedVideos()
+            deferredVideos = context!!.application.serviceLocator.pocket.getRecommendedVideos()
```

Comment:
Can this just be `context!!.serviceLocator`, because you've added an ext to `Context` already?